### DISCUSSION TITLE
Is it possible to hide the "(X hidden items)" text?

### DISCUSSION THREAD

**Discussion by nvsd** [nvim-neo-tree/neo-tree.nvim#750] [Q&A] [answered]
It's not helpful to see "(1 hidden item)" for things like DS_Store.

In general, it would be nice to hide these selectively. For example, ignore the text if it's node_modules, DS_Store, .git, and things like that.

**Comment by cseickel** [comment, marked as answer]
Setting this to false should remove the message:
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/78bb9963ddf0d664bcce40da08093983a861ed16/lua/neo-tree/defaults.lua#L414
